src/renderer/components/dashboardPage/issues/index.vue

```diff
@@ -110,6 +118,15 @@ export default {
           this.fetchStatusesForProject()
         }
       }
+    },
+    getFilteredIssues: {
+      immediate: true,
+      handler () {
+        this.noIssues = false
```

Comment:
OK, I see why there is the tricky solution with `noIssues` flag instead of the computed property, but the problem should be resolved at the source, not in a template.

@grzch:
There are [two actions](https://github.com/SparingSoftware/workometer/blob/develop/src/renderer/store/modules/issues.js#L44-L55) to fill the store with issues: `fetchIssuesForSprint` and `fetchIssuesForBoard` for backlogs. But both of them share **the same loading state** (`'issuesLoading'`). 
If you quickly change between sprint and backlog, two methods are running at the same time. The winner of this race inform the whole application that the issues fetching is done, even if the second action has not finished yet.